torch/overrides.py

```diff
@@ -332,6 +332,7 @@ def get_testing_overrides() -> Dict[Callable, Callable]:
     # See Issue #28233.
     Tensor = torch.Tensor
     ret: Dict[Callable, Callable] = {
+        torch.nn.functional.bias: lambda input, bias: -1,
```

Comment:
not totally sure if this is the right/only one one needed here

Reply:
I think this is it; it's the only function defined on the Python side